package.json

```diff
@@ -18,7 +18,7 @@
   "dependencies": {
     "@types/node-cron": "^2.0.3",
     "discord.js": "^12.5.1",
-    "dotenv": "^6.1.0",
+    "dotenv": "^10.0.0",
```

Comment:
Note: This may seem like it's not doing much, but v6 doesn't include types so it actually errors the build in CI